src/components/Carousel/arrow.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import ArrowImage from '../../assets/Carousel/arrow.svg'
+
+const CarouselArrow = ({
+  direction,
+  className,
+  onClick,
+  isHidden,
+}) => {
+  const classNames = cn(
+    'mc-carousel-arrow',
+    { [className]: !!className },
```

Comment:
I always try to avoid `Boolean()` because of the difference between `new Boolean()` and `Boolean()`. I think it can be very hard to differentiate them when debugging.

But I saw you guys are using Boolean in all the code, so I'll do the same to keep it consistent.